src/main/java/net/dv8tion/jda/core/events/guild/voice/GuildVoiceJoinEvent.java

```diff
@@ -30,6 +35,9 @@ public GuildVoiceJoinEvent(JDA api, long responseNumber, Member member)
         this.channelJoined = member.getVoiceState().getChannel();
     }
 
+    /**
+     * @return the {@link net.dv8tion.jda.core.entities.VoiceChannel} that was joined
+     */
```

Comment:
Should probably also have a description on the method itself instead of only for return

Reply:
Like this?
![image](https://user-images.githubusercontent.com/2582617/31057519-525f3b1c-a6e4-11e7-9ef6-c9b9c0255b14.png)

Reply:
yea. but would prepend "The" or "Returns the"

Reply:
and also add to the return

Reply:
there should also be a empty line between description and return